GuildedRoseLLCSpec/ItemViewControllerSpec.swift

```diff
@@ -20,67 +20,62 @@ class ItemViewControllerSpec: QuickSpec {
             
             describe("viewDidLoad") {
                 it("creates a data source") {
+                    
                     controller.viewDidLoad()
                     
                     expect(controller.itemCollectionView.dataSource).notTo(beNil())
                 }
                 
                 it("creates a repository") {
+                    
                     controller.viewDidLoad()
                     
                     expect(controller.itemRepository).notTo(beNil())
                 }
                 
                 it("sets the data source to an empty list") {
-                    controller.viewDidLoad()
                     
-                    let dataSource = controller.itemCollectionView.dataSource as? ItemCollectionViewDataSource
+                    controller.viewDidLoad()
                     
-                    expect(dataSource?.items).to(equal([]))
+                    expect((controller.itemCollectionView.dataSource as? ItemCollectionViewDataSource)?.items).to(equal([]))
```

Comment:
I actually prefer having two lines to one in this scenario. Having a type cast inside of the `expect` is a bit hard to read.

Reply:
It is not needed here, but is on line 53, to ensure test sets datasource asynchronously (in the toEventually)